help/redirects/sqlserver.md

```diff
@@ -0,0 +1,6 @@
+# Packaging and Deploying SQL Databases
```

Comment:
I don't think we need a redirection page as it didn't exist previously. We mainly have those to not break old links.